api/src/test/java/org/pmiops/workbench/tools/cdrconfig/CdrConfigVOMapperTest.java

```diff
@@ -23,7 +24,7 @@
 import org.springframework.test.context.junit4.SpringRunner;
 
 @RunWith(SpringRunner.class)
-@Import(CdrConfigVOMapperImpl.class)
+@Import({CdrConfigVOMapperImpl.class, CommonConfig.class})
 @DataJpaTest
 @DirtiesContext(classMode = DirtiesContext.ClassMode.BEFORE_EACH_TEST_METHOD)
 public class CdrConfigVOMapperTest extends SpringTest {
```

Comment:
Withe the above change, it seems like there is no need to extend `SpringTest` anymore - can that be deleted?